setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='openreview-py',
-    version='1.2.3',
+    version='1.2.4',
```

Comment:
I discussed this with Celeste and we decided to increase the version number only when we are going to make a release in the instances of the live site. Please remove this change for this PR.